src/views/GithubCodePermalinkReferenceWidget.vue

```diff
@@ -181,6 +184,28 @@ export default {
 }
 </script>
 
+<!--style lang="scss">
+// does not work :-)
+body[data-theme-default] {
+	@media (prefers-color-scheme: dark) {
+		@import 'highlight.js/styles/github-dark-dimmed.css';
```

Comment:
Does not seem to work as the import matches the criteria from https://sass-lang.com/documentation/at-rules/import#plain-css-imports

Changing to this seems to do the trick though:

```suggestion
		@import 'highlight.js/styles/github-dark-dimmed';
```

Reply:
:pray: Thanks a lot. All good now.